src/App.js

```diff
@@ -1,13 +1,64 @@
 import React from "react";
-
+import Heading from "./Heading";
 import Bookings from "./Bookings";
 import "./App.css";
+import TouristInfoCards from "./TouristInfoCards";
+import Footer from "./Footer";
+import Restaurant from "./Restaurant";
+
+const info = [
+  "123 Fake Street, London, E1 4UD",
+  "hello@fakehotel.com",
+  "0123 456789"
+];
+
+const TouristInfoDetails = [
```

Comment:
I like that we are separating our data from our components, makes the components a lot simpler!

But it is good to use camelCase for our JS variables! This is starting with a capital